x-pack/plugins/triggers_actions_ui/public/common/constants/index.ts

```diff
@@ -12,3 +12,5 @@ export { builtInGroupByTypes } from './group_by_types';
 export const VIEW_LICENSE_OPTIONS_LINK = 'https://www.elastic.co/subscriptions';
 // TODO: Remove when cases connector is available across Kibana. Issue: https://github.com/elastic/kibana/issues/82502.
 export const DEFAULT_HIDDEN_ACTION_TYPES = ['.case'];
+
+export const PLUGIN_ID = 'triggersActionsUi';
```

Comment:
I think we should use the one defined here https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/plugin.ts#L143.

Reply:
I'll export it here and use it there. thanks for the tip